bindings/python/crocoddyl/core/solvers/ipopt/ipopt.cpp

```diff
@@ -0,0 +1,33 @@
+#include "python/crocoddyl/core/core.hpp"
+#include "crocoddyl/core/solvers/ipopt/ipopt.hpp"
+
```

Comment:
Do not create an unnecessary sub-folder.
Please move this file to
`bindings/python/crocoddyl/core/solvers/ipopt.cpp`